girder/stemserver_plugin/models/stemimage.py

```diff
@@ -108,11 +132,11 @@ def _stream():
 
         return _stream
 
-    def bright(self, id, user):
-        return self._get_h5_dataset(id, user, '/stem/bright')
+    def bright(self, id, format, user):
+        return self._get_h5_dataset(id, format, user, '/stem/bright')
```

Comment:
As these are 'full' arrays and we are now using MessagePack, let go head and return the 2d array, so the client doesn't have to get the size if they are using the MessagePack format.

Reply:
@cjh1 Just for clarification, do we want to use msgpack by default now and not use bytes at all? Or keep using bytes?

This is currently returning a full 2D array using msgpack, although I might want to change the format of a few things here once we get #8 merged.

Reply:
Not sure on what our default should be, but it would be good to support both options in my opinion if it isn't that hard to do so - doesn't look like it is.

Reply:
Yes, @psavery an I just chatted and we will support both for now.